gis/librttopo/Portfile

```diff
@@ -5,7 +5,7 @@ PortSystem          1.0
 name                librttopo
 categories          gis
 license             GPL-2
-version             1.1.0-RC1
+version             1.1.0
```

Comment:
I suspect that this may not be recognized as a new version and the `epoch` will need to be added and set to 1.

Reply:
Thanks @mf2k, you were correct. I added the new epoch.